src/common/components/clusterWizard/networkingSteps/NetworkTypeControlGroup.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Split, SplitItem, Tooltip } from '@patternfly/react-core';
+import { RadioField } from '../../ui/formik';
+import { PopoverIcon } from '../../../components/ui';
+
+export interface NetworkTypeControlGroupProps {
+  isIPv6: boolean;
+}
+
+export const NetworkTypeControlGroup = ({ isIPv6 = false }: NetworkTypeControlGroupProps) => {
```

Comment:
I prefer to be consistent with typing of our components
```suggestion
export const NetworkTypeControlGroup: React.FC<NetworkTypeControlGroupProps> = ({ isIPv6 = false }) => {
```